### ISSUE TITLE
Face loss for face reconstruction

### ISSUE THREAD

**Issue by kl2004** [TencentARC/SEED-Voken#31]
Thanks for sharing the code!

I tried the checkpoint [IN256_Base](https://huggingface.co/TencentARC/Open-MAGVIT2/blob/main/imagenet_256_B.ckpt) on face images, but the faces in the reconstructed images are not great. I wonder how the reconstructed faces could be improved. Do you plan to include face loss in the total loss?

**Comment by vinyesm** [comment]
Same observation, here an example with imagenet_256_L. But I think it could be related to training data
<img width="611" alt="Screenshot 2024-09-09 at 17 49 15" src="https://github.com/user-attachments/assets/4d9ec8a4-2333-4044-b65c-a71e0eb12098">

**Comment by RobertLuo1** [comment]
Hi, there, since the face image is hard to collect and the current scale of face images is limited. Considering the downsampling rate which also causes difficulty in accurately reconstructing the small detailed object such as face. 
By the way, we observe that including more training data will benefit which can be reflected in our pretrain tokenizer version.
